test/flutter_mailer_e2e.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter_mailer/flutter_mailer.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:e2e/e2e.dart';
+
+ void main() {
+   E2EWidgetsFlutterBinding.ensureInitialized();
```

Comment:
test/ should not be a E2E but a unit test (for the method channel) only. Something like this: https://github.com/flutter/plugins/blob/master/packages/battery/test/battery_test.dart#L13

So you mock the channel and test that the Dart code calls the right methods on the channel. In the e2e test you test the actual call on the platform.